### PR TITLE
We need to dispatch onto the main queue before we do anything in the AVS handlers

### PR DESCRIPTION
**Pull request by typfel**
This fixes an issue where the outgoing video was missing when accepting a call.

https://wearezeta.atlassian.net/browse/ZIOS-8400

### CODE REVIEW

**Comment by typfel** [comment]
This also fixes https://github.com/wireapp/wire-ios-sync-engine/pull/341

**Comment by mikeger** [comment]
Before AVS used to synchronize everything on their own, changed with the new API?

**Comment by typfel** [comment]
@mikeger AVS still marshals the calls to the correct thread when we mute the audio for example. But these are callback handlers and they called on the AVS thread, they expect us dispatch away immediately from that thread. Calling any AVS functions for that thread may result in a deadlock.

File: Source/Calling/WireCallCenterV3.swift [hunk 1]
```diff
@@ -339,22 +359,21 @@ public typealias WireCallMessageToken = UnsafeMutableRawPointer
...
+            establishedDate = Date()
+            
+            if avsWrapper.isVideoCall(conversationId: conversationId) {
+                avsWrapper.setVideoSendActive(userId: conversationId, active: true)
             }
-        default:
-            callState.postNotificationOnMain(conversationID: conversationId, userID: userId, uiMOC: uiMOC)
         }
+    
+        callState.postNotificationOnMain(conversationID: conversationId, userID: userId, uiMOC: uiMOC)
```
Comment (basine) [inline Source/Calling/WireCallCenterV3.swift]: We wrap the posting of the notification in `postNotificationOnMain` in another performGroupedBlock. I guess that can go then as well.

**Pull request merged**